sdk/typescript/test/e2e/utils/setup.ts

```diff
@@ -68,9 +70,15 @@ export function getClient(url = DEFAULT_FULLNODE_URL): SuiClient {
 export async function setup(options: { graphQLURL?: string; rpcURL?: string } = {}) {
 	const keypair = Ed25519Keypair.generate();
 	const address = keypair.getPublicKey().toSuiAddress();
+	setupClientConfig();
 	return setupWithFundedAddress(keypair, address, options);
 }
 
+export async function setupClientConfig() {
+	const configPath = path.join(homedir(), '.sui', 'sui_config', 'client.yaml');
+	execSync(`${SUI_BIN} client --yes --client.config ${configPath}`, { encoding: 'utf-8' });
```

Comment:
Setting up global configs as part of the the typescript e2e tests seems like a bad idea.  I don't think we want to use a users global configs when running e2e tests locally.  If this is just for CI, it's better to handle this setup as part of the github action template